mlrun/model_monitoring/applications/base.py

```diff
@@ -271,7 +272,12 @@ def _handler(
                 "working with endpoints, without any custom data-frame input"
             )
 
-        application_name = self.__class__.__name__
+        if not application_name:
+            application_name = self.__class__.__name__
```

Comment:
Also I think we can add some log to inform the user what is the name no?

Reply:
Added info log in the `write_output=True` case. Otherwise, the `application_name` is not very relevant